tensorflow/core/kernels/cwise_ops_test.cc

```diff
@@ -49,21 +41,23 @@ static int RowsAndColsArg(int r, int c) { return r * kRows + c; }
 static int RowsFromArg(int arg) { return (arg / kRows); }
 static int ColsFromArg(int arg) { return (arg % kRows); }
 
-#define BM_UNARY(DEVICE, FUNC, TYPE)                              \
-  static void BM_##DEVICE##_##FUNC_##TYPE(int iters, int num) {   \
-    const int64 tot = static_cast<int64>(iters) * num;            \
-    testing::ItemsProcessed(tot);                                 \
-    testing::BytesProcessed(tot * sizeof(float));                 \
-    test::Benchmark(#DEVICE, Unary(#FUNC, num, TYPE)).Run(iters); \
-  }                                                               \
+#define BM_UNARY(DEVICE, FUNC, T, TYPE)                              \
+  static void BM_##DEVICE##_##FUNC_##TYPE(int iters, int num) {      \
+    const int64 tot = static_cast<int64>(iters) * num;               \
+    testing::ItemsProcessed(tot);                                    \
+    testing::BytesProcessed(tot * sizeof(float));                    \
```

Comment:
Yea, I think this wouldn't be covered by any test -- it's simply used to report the number of bytes processed (for calculating throughput of the benchmark I think?). 

Annoyingly, the benchmark isn't run at all by the test suite! You have to build the test binary then run the test binary with the ```--benchmarks=all``` flag.

Something like:
```
$ bazel build //tensorflow/core/kernels:cwise_ops_test
$ ./bazel-bin/tensorflow/core/kernels/cwise_ops_test --benchmarks=all
```